### PR TITLE
Check the validity of a CSV file.

### PR DESCRIPTION
**Pull request by HeartPattern**
Check the existence of students.csv file before reading it.

### CODE REVIEW

File: src/Server/DB.fs [hunk 1]
```diff
@@ -44,6 +44,10 @@ let private parseLine (line: string) =
   arr.[0].Split(' ') |> Seq.last |> (fun s -> s.ToLower ()),
   arr.[1]
 
+let assertDBExistence dbpath =
+  if File.Exists dbpath then ()
+  else failwith "Cannot reload: DB doesn't exist."
```
Comment (sangkilc) [inline src/Server/DB.fs]: Now that this function is used in another context, the message is wrong.

File: src/Server/DB.fs [hunk 2]
```diff
@@ -54,8 +58,9 @@ let private initSessionDir stime =
   |> fun info -> info.FullName
 
 let private initStudents () =
-  Path.Combine (dbdir, studentfile)
-  |> File.ReadLines
+  let dir = Path.Combine (dbdir, studentfile)
+  assertDBExistence dir
```
Comment (sangkilc) [inline src/Server/DB.fs]: Although there's no big difference in this particular case, showing a well-formed error message is a good practice.

File: src/Server/DB.fs (hunk 1 above)
Comment (sangkilc) [reply src/Server/DB.fs]: This function should not be moved since it is not called by initDBDir.

**Review by sangkilc — CHANGES_REQUESTED** [review]
Your final modification should be only about Line 61-64.

**Comment by sangkilc** [comment]
Thanks for your contribution. What's your name?

**Comment by HeartPattern** [comment]
My name is 홍연준 (student id 20190698)

**Pull request merged**